tests/e2e/e2e_ibc_test.go

```diff
@@ -118,49 +111,56 @@ func (s *E2ETest) TestIBCTokenTransfer() {
 			sdk.NewDecFromInt(tokenQuota).Quo(atomPrice).Mul(powerReduction).RoundInt(),
 		)
 
+		//<<<< INFLOW : gaia -> umee >>
 		// send $500 ATOM from gaia to umee. (ibc_quota will not check token limit)
 		atomFromGaia := mulCoin(atomQuota, "5.0")
 		atomFromGaia.Denom = "uatom"
 		s.SendIBC(setup.GaiaChainID, s.Chain.ID, "", atomFromGaia, false, "")
 		s.checkSupply(umeeAPIEndpoint, uatomIBCHash, atomFromGaia.Amount)
 
+		// <<< OUTLOW : umee -> gaia >>
 		// compute the amout of UMEE sent to gaia which would meet umee's token quota
 		umeePrice, err := s.QueryHistAvgPrice(umeeAPIEndpoint, umeeSymbol)
 		s.Require().NoError(err)
 		s.Require().True(umeePrice.GT(sdk.MustNewDecFromStr("0.001")),
 			"umee price should be non zero, and expecting higher than 0.001, got: %s", umeePrice)
 		umeeQuota := sdk.NewCoin(appparams.BondDenom,
-			sdk.NewDecFromInt(tokenQuota).Quo(atomPrice).Mul(powerReduction).RoundInt(),
+			sdk.NewDecFromInt(tokenQuota).Quo(umeePrice).Mul(powerReduction).RoundInt(),
 		)
 
-		// send $90 UMEE from umee to gaia (ibc_quota will check)
-		// Note: receiver is null so hermes will default send to key_name (from config) of target chain (gaia)
-		sendUmee := mulCoin(umeeQuota, "0.9")
-		s.SendIBC(s.Chain.ID, setup.GaiaChainID, "", sendUmee, false, fmt.Sprintf(
-			"sending %s (less than token quota) ", sendUmee.String()))
-		s.checkOutflows(umeeAPIEndpoint, appparams.BondDenom, true, sdk.NewDecFromInt(sendUmee.Amount), appparams.Name)
-		s.checkSupply(gaiaAPIEndpoint, umeeIBCHash, sendUmee.Amount)
+		// << TOKEN QUOTA EXCCEED >>
+		// send $110 UMEE from umee to gaia (token_quota is 100$)
+		exceedUmee := mulCoin(umeeQuota, "1.1")
```

Comment:
why do we change the order?

Reply:
just making alignment like atom and umee sendIBC are execeedQuota , and It doesn't effect the flow of checking quota